backend/handler/scan_handler.py

```diff
@@ -183,6 +193,9 @@ async def scan_rom(
         "name": fs_rom["file_name"],
         "url_cover": "",
         "url_screenshots": [],
+        "crc_hash": rom.crc_hash if rom else None,
+        "md5_hash": rom.md5_hash if rom else None,
+        "sha1_hash": rom.sha1_hash if rom else None,
```

Comment:
why was this added here? it's usually pulled in later in the scan process

Reply:
honestly do not remeber it, removed